content/en/guides/core/artifacts/download-and-use-an-artifact.md

```diff
@@ -14,6 +14,18 @@ Download and use an artifact that is already stored on the W&B server or constru
 Team members with view-only seats cannot download artifacts.
 {{% /alert %}}
 
+{{% alert color="warning" %}}
+Artifacts created with reference URIs using `checksum=False` cannot be downloaded. To ensure your artifacts can be downloaded, use `checksum=True` (the default) when adding references:
```

Comment:
```suggestion
If you set `checksum=False` when adding a reference, the artifact cannot be downloaded later. Only use `checksum=False` when you need to track reference URIs without downloading capability. For downloadable artifacts, always use `checksum=True` (the default):
```
This way the notes at least share most of the text. But if checksum=True is the default, then line 22 seems not to need the `checksum=True` and it might be more clear to leave it off.